code/api/src/setup/database.js

```diff
@@ -16,15 +16,15 @@ const connection = new Sequelize(databaseConfigEnv.database, databaseConfigEnv.u
 })
 
 // Test connection
-console.info('SETUP - Connecting database...')
-
-connection
-  .authenticate()
-  .then(() => {
-    console.info('INFO - Database connected.')
-  })
-  .catch(err => {
-    console.error('ERROR - Unable to connect to the database:', err)
-  })
+// console.info('SETUP - Connecting database...')
```

Comment:
This was causing errors with the tests. As discussed in our testing class, it is okay to comment out